openassessment/fileupload/backends/swift.py

```diff
@@ -63,6 +63,10 @@ def get_download_url(self, key):
             )
             raise FileUploadInternalError(ex)
 
+    def remove_file(self, key):
+        """
+        @TODO Need to be implemented to remove previously uploaded files
+        """
```

Comment:
@efischer19 Seems that I've found a solution. I've configured [virtualbox with swift](https://github.com/swiftstack/vagrant-swift-all-in-one) and this code could be used to remove files on swift storage:

```
temp_url = swiftclient.utils.generate_temp_url(
    path='%s/%s/%s' % (url.path, bucket_name, key_name),
    key=key,
    method='DELETE',
    seconds=3600)
remove_url = '%s://%s%s' % (url.scheme, url.netloc, temp_url)
response = requests.delete(remove_url)
```